docs/content/reference/exploiters/SNMP.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "SNMP"
+date: 2023-05-15T12:53:22+00:00
+draft: false
+tags: ["exploit", "linux"]
+---
+
+The SNMP exploiter exploits [CVE-2020-15862](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-15862).
+
+
+### Description
+
+In versions 1 and 2c of the SNMP protocol the only means of authentication is the community string. In these versions of SNMP, community strings are sent as plaintext in all requests. If a community is configured with read-write access, the machine is vulnerable to an attack using NET-SNMP-EXTEND-MIB to run arbitrary commands on the victim.
+
```

Comment:
v3 is still vulnerable to CVE2020-15862, right? It's just that authenticating is harder. This language makes it seem like v3 is not vulnerable.

After clarifying the above, we may want to add an explicit note regarding which protocol versions we support in this exploiter.